core/math/plane.h

```diff
@@ -37,7 +37,7 @@ class Variant;
 
 struct _NO_DISCARD_ Plane {
 	Vector3 normal;
-	real_t d = 0;
+	real_t d;
```

Comment:
I agree. In fact, in #90866 I missed the fact that `Type type;` in `Variant` had no class-level initializer. Otherwise, I had not approved it. My bad. Can you please re-add it (maybe in this PR)? I agree with moving extra initializers from regular sentences in constructors to initializer lists in cases like these, but the "basic" default values should still be in the declaration of members so, as @lawnjelly says, you always have correct values no matter what and so you can trust that a defaulted constructor will yield no garbage.